package-lock.json

```diff
@@ -1549,9 +1657,9 @@
       }
     },
     "node_modules/@mongodb-js/saslprep": {
-      "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/@mongodb-js/saslprep/-/saslprep-1.1.0.tgz",
-      "integrity": "sha512-Xfijy7HvfzzqiOAhAepF4SGN5e9leLkMvg/OPOF97XemjfVCYN/oWa75wnkc6mltMSTwY+XlbhWgUOJmkFspSw==",
+      "version": "1.1.4",
```

Comment:
Saslprep is still upgraded, this should go back to the version it is on main. 

To be sure there aren't other accidental updates can you: 

- revert package-lock.json to be the same as main 
- rm -rf node_modules, 
- run npm install

then the `package-lock.json` should be upgraded to what we've modified in the package.json

Reply:
Ah I see, I was missing the `rm -rf node_modules`, thanks for the reminder!